### DISCUSSION TITLE
Possible to achieve something like this?

### DISCUSSION THREAD

**Discussion by gerroon** [tin2tin/Pallaidium#77] [Q&A] [answered]
Hi

Is it possible to do something like this with Palladium? Video based generation

https://x.com/aianddesign/status/1719117917915140357?s=46

**Comment by tin2tin** [comment, marked as answer]
Pallaidium do have video to video. So, try to add a short video(duration depending on your vram), select Strips as input, select the video, select Video & Zeroscope XL as output and play around with the prompt and various values.

**Reply by gerroon** [reply]
What duration ranges do your recommend? I have 64gb ram and 12gb gpu ram.

**Comment by tin2tin** [comment]
I'm on 6 GB VRAM, so I wouldn't know. But if you find your sweet spot on your hardware, maybe you could share it, so other users can use that information?
